src/commands/synthetics/run-tests-lib.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from './interfaces'
 import {DefaultReporter, getTunnelReporter} from './reporters/default'
 import {JUnitReporter} from './reporters/junit'
-import {DEFAULT_BATCH_TIMEOUT, DEFAULT_COMMAND_CONFIG} from './run-tests-command'
+import {DEFAULT_BATCH_TIMEOUT, DEFAULT_COMMAND_CONFIG, DEFAULT_TEST_CONFIG_FILES_GLOB} from './run-tests-command'
```

Comment:
I don't see `DEFAULT_TEST_CONFIG_FILES_GLOB` being now used in this file, so i assume this is a leftover from a previous change 🤔 

```suggestion
import {DEFAULT_BATCH_TIMEOUT, DEFAULT_COMMAND_CONFIG} from './run-tests-command'
```